src/lparser.js

```diff
@@ -640,8 +640,7 @@ const open_func = function(ls, fs, bl) {
     fs.nactvar = 0;
     fs.firstlocal = ls.dyd.actvar.n;
     fs.bl = null;
-    let f = new Proto(ls.L);
-    f = fs.f;
+    let f = fs.f;
```

Comment:
We don't need a new object here ?

Reply:
It was immediately thrown away. `fs.f` already contains a `Proto`